### PR TITLE
#85 - make toast notification

### LINKED ISSUE #85: Make toast notification
The aim of this task is to add toast notifications. We can use [Vue Toastification](https://vue-toastification.maronato.dev/) 

         
              I think we should implement toast, but in another task.

_Originally posted by @EwelinaLasowy in https://github.com/blumilksoftware/escooters/pull/67#discussion_r1276079649_

### PR DESCRIPTION
**Pull request by BarTracz**
This should close #85

### CODE REVIEW

File: lang/pl.json [hunk 1]
```diff
@@ -67,13 +67,34 @@
     "Open main menu": "Otwórz menu główne",
     "Please verify your credentials and try again": "Sprawdź swoje dane logowania i spróbuj ponownie",
     "Invalid password or username": "Nieprawidłowe hasło lub nazwa użytkownika",
-
+    "Show favorites": "Pokaż ulubione",
+    "City added to favorites!": "Miasto dodane do ulubionych!",
```
Comment (EwelinaLasowy) [inline lang/pl.json:71]: I think I will remove exclamation mark at the end of the sentence in all toast notifications.

File: lang/pl.json [hunk 2]
```diff
@@ -67,13 +67,34 @@
...
+    "Country created successfully!": "Kraj utworzony pomyślnie!",
+    "City deleted successfully!": "Miasto usunięte pomyślnie!",
+    "Country deleted successfully!": "Kraj usunięty pomyślnie!",
+    "Login to add to favorites!": "Zaloguj się, aby dodać do ulubionych",
+    "There was an error creating city!": "Wystąpił błąd podczas tworzenia miasta!",
+    "There was an error creating country!": "Wystąpił błąd podczas tworzenia kraju!",
+    "There was an error adding alternative city name!": "Wystąpił błąd podczas dodawania alternatywnej nazwy miasta!",
+    "Alternative city name added successfully!": "Alternatywna nazwa miasta dodana pomyślnie!",
+    "Alternative city name deleted successfully!": "Alternatywna nazwa miasta usunięta pomyślnie!",
+    "City providers updated successfully!": "Dostawcy miasta zaktualizowani pomyślnie!",
```
Comment (EwelinaLasowy) [inline lang/pl.json:85]: "Dostawcy miasta"?  Can we phrase it differently?

File: lang/pl.json (hunk 1 above)
Comment (BarTracz) [reply lang/pl.json]: fixed

File: lang/pl.json (hunk 2 above)
Comment (BarTracz) [reply lang/pl.json]: fixed

File: lang/pl.json [hunk 3]
```diff
@@ -67,17 +67,40 @@
     "Open main menu": "Otwórz menu główne",
     "Please verify your credentials and try again": "Sprawdź swoje dane logowania i spróbuj ponownie",
     "Invalid password or username": "Nieprawidłowe hasło lub nazwa użytkownika",
-
+    "Show favorites": "Pokaż ulubione",
+    "City added to favorites": "Miasto dodane do ulubionych",
```
Comment (EwelinaLasowy) [inline lang/pl.json:71]: Please add a dot at the end of the sentences.

**Review by EwelinaLasowy — APPROVED** [review]

**Review by AleksandraKozubal — APPROVED** [review]

**Comment by kamilpiech97** [comment]
js linter @BarTracz

**Comment by mtracz** [comment]
Conflicts.

**Review by mtracz — APPROVED** [review]

**Pull request merged**